static/src/js/schemas/uiSchemas/services/service_options.js

```diff
@@ -0,0 +1,286 @@
+const serviceOptionsUiSchema = {
+  'ui:submitButtonOptions': {
+    norender: true
+  },
+  ServiceOptions: {
+    'ui:heading-level': 'h3', // TODO: see why this title is not being effected.
+    'ui:field': 'layout',
+    'ui:layout_grid': {
+      'ui:row': [
+        {
+          'ui:col': {
+            md: 12,
+            children: [
+              {
+                'ui:group': 'Service Options',
+                'ui:group-description': true,
+                'ui:row': [
+                  {
+                    'ui:col': {
+                      md: 12,
+                      children: ['Subset']
+                    }
+                  },
+                  {
+                    'ui:col': {
+                      md: 12,
+                      children: ['VariableAggregationSupportedMethods']
+                    }
+                  },
+                  {
+                    'ui:col': {
+                      md: 12,
+                      children: ['SupportedInputProjections']
+                    }
+                  },
+                  {
+                    'ui:col': {
+                      md: 12,
+                      children: ['SupportedOutputProjections']
+                    }
+                  },
+                  {
+                    'ui:col': {
+                      md: 12,
+                      children: ['InterpolationTypes']
+                    }
+                  },
+                  {
+                    'ui:col': {
+                      md: 12,
+                      children: ['SupportedReformattings']
+                    }
+                  },
+                  {
+                    'ui:col': {
+                      md: 12,
+                      children: ['MaxGranules']
+                    }
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      ]
+    },
+    SupportedReformattings: {
+      items: {
+        'ui:field': 'layout',
+        'ui:layout_grid': {
+          'ui:row': [
+            {
+              'ui:col': {
+                md: 12,
+                children: [
+                  {
+                    'ui:row': [
+                      {
+                        'ui:col': {
+                          md: 12,
+                          children: ['SupportedInputFormat']
+                        }
+                      },
+                      {
+                        'ui:col': {
+                          md: 12,
+                          children: ['SupportedOutputFormats']
+                        }
```

Comment:
Is there any way to remove this header and put the description underneath where it says "Supported Output Formats (1 of 1)? It's a little wonky as is